kft-2020/README.md

```diff
@@ -0,0 +1,19 @@
+# Kubernetes Fest Tokyo 2020: Kubernetes Upstream Training
+
+[<img src="https://github.com/kubernetes/kubernetes/raw/master/logo/logo.png" width="64">](https://github.com/kubernetes/kubernetes)
+[<img src="./images/xxx.png" width="64">](xxx)
```

Comment:
I'm not sure to understand but if you want to display the image you need to add it to the PR. 
The Markdown code will be : 
```
![xxx](images/xxx.png)
```

Reply:
Apparently the image doesn't exist yet.  But since it doesn't, it should be commented out.

Also ... I don't think that's valid markdown.

Reply:
@ameukam @jberkus Thank you to comments!

**1. about "xxx"**

Yes, "xxx" is just a placeholder which doesn't exist, but it's hard to understand.
I'll comment out items which doesn't exist.

**2. markdown format to display images with hyperlink**

I change this section to display "plain image" and "event title with hyperlink".